extra-dev/packages/coq-stdpp/coq-stdpp.dev/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "Ralf Jung <jung@mpi-sws.org>"
```

Comment:
I would appreciate if this package would be removed. We deliberately did not put a coq-stdpp.dev package into this repository. The 'dev' version is meant for local pinning only.

I am surprised that the people maintaining this software (and supposedly even maintaining this package!) did not get pinged...

Reply:
Note that this is not the first time this happened, there were "rogue" coq-iris.dev packages already in 2017 and they have been removed again (https://github.com/coq/opam-coq-archive/issues/191 -- though those versions really made no sense, this time at least the dev packages are actually properly tracking git).

Reply:
https://github.com/coq/opam-coq-archive/pull/2256 proposes to remove these packages again.